.env.development

```diff
@@ -1,2 +1,8 @@
 AUTH_SECRET=secret
-REDIRECT_URI=http://localhost:3000/auth-redirect
+DB_PORT=5432
+DB_USER=postgres
+DB_HOST=localhost
+DB_NAME=dev-directory
+DISCORD_GUILD_ID=735923219315425401
+DISCORD_REDIRECT_URI=http://localhost:3000/auth-redirect
+NEXT_PUBLIC_DISCORD_REDIRECT_URI=http://localhost:3000/auth-redirect
```

Comment:
Environment variables don't typically get loaded into the client because it's not safe.  We can prefix them with `NEXT_PUBLIC_` to make our client bundler load them in.  See more [here](https://nextjs.org/docs/basic-features/environment-variables#exposing-environment-variables-to-the-browser).

Reply:
Bit late, as I see it was merged... but, since the trailing path on the `REDIRECT_URI`s is fixed and won't change per installation, could we simplify the env by replacing them with a single, more generic `BASE_URL`?
```js
`${process.env.BASE_URL}/auth-redirect`
```

Client-side we can create a simple function to build it using `window.locaion`. eg.
```js
const baseUrl = () => {
    const { protocol, hostname, port } = window.location;
    return `${protocol}//${hostname}${port && `:${port}`}`;
};
```

Also, perhaps a more conventional endpoint URI instead of `/auth-redirect`, eg.
```js
`${baseUrl()}/auth/callback`

// or auth/[provider]/callback
`${baseUrl()}/auth/discord/callback`
```

Reply:
Good points! I would agree if this weren't also a config that we have to set on the the Discord developer dashboard: 
![image](https://user-images.githubusercontent.com/5304277/224204376-7ad3f473-965b-4a7b-bdf1-acd402edef02.png)

Given that, I think it's generally easier to keep as one big string rather than a computed value to keep it in sync with the discord dashboard

Reply:
Ahh, that's fair. 👍🏼 I suppose I was viewing it the other way around... as if the code would be the authoritative source of truth and you'd change Discord to match.